yb_migrate/src/migration/postgres.go

```diff
@@ -67,15 +66,18 @@ func PgDumpExtractSchema(source *utils.Source, exportDir string) {
 	//Parsing the single file to generate multiple database object files
 	parseSchemaFile(source, exportDir)
 
-	// utils.PrintIfTrue("export of schema done!!!", !source.GenerateReportMode)
+	if source.GenerateReportMode {
+		log.Infoln("Scanning of schema completed.")
+	} else {
+		log.Infoln("Export of schema completed.")
+	}
 	utils.WaitChannel <- 0
 	<-utils.WaitChannel
 }
 
 //NOTE: This is for case when --schema-only option is provided with pg_dump[Data shouldn't be there]
```

Comment:
minor extra-work(optional): add space if  i.e. "// NOTE: ..." similarly for other comments